CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Features
 - Added sourcemaps to the dist output to simplify debugging @miroslavstastny ([#2329](https://github.com/microsoft/fluent-ui-react/pull/2329))
+- Added virtualized table prototype using `react-virtualized` and `Table`components @pompomon ([#2339]https://github.com/microsoft/fluent-ui-react/pull/2339)
```

Comment:
... `([#2339](https://github.com/microsoft/fluent-ui-react/pull/2339))` - link in parentheses.